libs/SalesforceHybrid/src/com/salesforce/androidsdk/phonegap/app/SalesforceHybridSDKManager.java

```diff
@@ -153,4 +179,58 @@ public String getAppType() {
 	public boolean isHybrid() {
 		return true;
 	}
+
+	/**
+	 * Setup global store using config found in assets/www/globalstore.json
+	 */
+	public void setupGlobalStoreFromDefaultConfig() {
+		SmartStoreLogger.d(TAG, "Setting up global store using config found in " + ConfigAssetPath.globalStore.path);
+		setupStoreFromConfig(getGlobalSmartStore(), ConfigAssetPath.globalStore.path);
+	}
+
+	/**
+	 * Setup user store using config found in assets/www/userstore.json
+	 */
+	public void setupUserStoreFromDefaultConfig() {
+		SmartStoreLogger.d(TAG, "Setting up user store using config found in " + ConfigAssetPath.userStore.path);
+		setupStoreFromConfig(getSmartStore(), ConfigAssetPath.userStore.path);
+	}
+
+    /**
+	 * Setup given store using config found in given json assets file
+	 *
+	 * @param store
+	 * @param assetPath
+	 */
+	private void setupStoreFromConfig(SmartStore store, String assetPath) {
+		StoreConfig config = new StoreConfig(context, assetPath);
+		config.registerSoups(store);
+	}
+
+    /**
+     * Setup global syncs using config found in assets/www/globalsyncs.json
+     */
+    public void setupGlobalSyncsFromDefaultConfig() {
```

Comment:
Overriding the method from SmartSyncSDKManager to read from an asset rather than a resource